crates/hdk3/src/guest_callback/entry_defs.rs

```diff
@@ -84,6 +84,20 @@ macro_rules! entry_def {
             }
         }
 
+        impl TryFrom<&$t> for $crate::prelude::Entry {
+            type Error = $crate::prelude::HdkError;
+            fn try_from(t: &$t) -> Result<Self, Self::Error> {
+                Ok(Self::App(SerializedBytes::try_from(t)?.try_into()?))
+            }
+        }
+
+        impl TryFrom<$t> for $crate::prelude::Entry {
+            type Error = $crate::prelude::HdkError;
+            fn try_from(t: $t) -> Result<Self, Self::Error> {
+                Self::try_from(&t)
```

Comment:
What happens if they already have an`Entry` does this result in recursively serialising it?

Reply:
@freesig no because you don't use `entry_defs!` macro on `Entry`